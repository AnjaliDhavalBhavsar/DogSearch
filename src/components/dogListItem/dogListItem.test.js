import TestRenderer from "react-test-renderer";
import DogDataProvider from "../../context";
import { DogListItem } from ".";

const data = {
  dogList: [
    {
      imperial: "23 - 25",
      height: "58 - 64",
      id: 9,
      name: "Alaskan Malamute",
      group: "Working",
      lifespan: "12 - 15 years",
      temperament: "Friendly, Affectionate, Devoted, Loyal, Dignified, Playful",
    },
    {
      imperial: "20 - 22",
      height: "58 - 64",
      id: 10,
      name: "Alaskan ",
      group: "Working",
      lifespan: "12 - 15 years",
      temperament: "Friendly, Affectionate, Devoted, Loyal, Dignified, Playful",
    },
  ],
};

test("DogList snapshot", () => {
  const element = new TestRenderer.create(
    (
      <DogDataProvider value={data}>
        <DogListItem dogDetail={data?.dogList[0]} />
      </DogDataProvider>
    )
  );
  expect(element).toMatchSnapshot();
});
