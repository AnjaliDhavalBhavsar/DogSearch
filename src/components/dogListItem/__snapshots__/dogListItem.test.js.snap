// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogList snapshot 1`] = `
<div
  className="detailContainer"
>
  <div
    className="imageContainer"
  >
    <img
      alt="NoImage"
      className="imageClass"
      height="100px"
      src="noImage.png"
      width="100px"
    />
  </div>
  <div
    className="column"
  >
    <div
      className="name"
    >
      <span
        className="bold"
      >
        Name:
      </span>
       
      Alaskan Malamute
    </div>
    <div
      className="name"
    >
      <span
        className="bold"
      >
        Temperament:
      </span>
       
      Friendly, Affectionate, Devoted, Loyal, Dignified, Playful
    </div>
  </div>
  <div
    className="column"
  >
    <div
      className="name"
    >
      <span
        className="bold"
      >
        Metric: 
      </span>
      58 - 64
    </div>
    <div
      className="name"
    >
      <span
        className="bold"
      >
        Imperial: 
      </span>
      23 - 25
    </div>
  </div>
  <div
    className="column flex"
  >
    12 - 15 years
  </div>
  <div
    className="column flex"
  >
    Working
  </div>
</div>
`;
